tests/page-objects/default/analytics/analytics.wdio.page.js

```diff
@@ -19,6 +19,20 @@ const targetNumberPercent = (targetElement) => targetElement.$('.body .target-pr
 const targetNumberPercentCount = (targetElement) => targetElement.$('.body .target-progress .number span:nth-child(2)');
 
 const targetGoalValue = (targetElement) => targetElement.$('.body .count .goal p');
+const AGGREGATE_LIST = '#target-aggregates-list';
+const loadingStatus = () => $(`${AGGREGATE_LIST} .loading-status`);
+const aggregateList = () => $$(`${AGGREGATE_LIST}  ul li`);
+const targetAggregateListItem = (contactId) => $(`.aggregate-detail li[data-record-id="${contactId}"] a`);
+
+const labels = () => $$('.content-pane .meta > div > .card .row label');
+const meta = () => $$('.content-pane .meta > div > .card .row p');
+const rows = () => $$('.content-pane .meta > div > .card .row p');
+const pane = () => $$('.content-pane .meta > div > .card .row label');
```

Comment:
Can you please update these selectors to something more specific? Maybe adding a `test-id` will be a good idea.